src/assets/stylesheets/avatar-selector.scss

```diff
@@ -0,0 +1,47 @@
+@import 'fonts';
+@import 'shared';
+
+#selector-root {
+  height: 100%;
+}
+.avatar-selector {
+  overflow: hidden;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+
+  &__prev-button:focus, &__next-button:focus {
```

Comment:
+1 gotta refactor my scss at some point

Reply:
When we do we might also want to switch to using `:local` styles and importing in js instead of using css imports. This has the advantage of scoping all the css so we dont have to use really long names or worry about unintended cascading. Also, it ties the css to just the code using it, so it makes dead code removal easier because you don't have to guess what rules might be applying to what.